frontend-react/yarn.lock

```diff
@@ -3217,7 +3217,7 @@
     "@webassemblyjs/ast" "1.11.1"
     "@xtuc/long" "4.2.2"
 
-"@xmldom/xmldom@^0.8.3":
+"@xmldom/xmldom@0.8.6", "@xmldom/xmldom@^0.8.3":
```

Comment:
This can be removed from this changeset.  It's not directly related to your changes, and it should be rectified on `master` already 🙏